include/alpaka/core/Common.hpp

```diff
@@ -31,13 +31,21 @@
     #include <intrin.h>
 #endif
 
+//-----------------------------------------------------------------------------
+// BOOST_PREDEF_MAKE_10_VVRRP(V)
+#if !defined(BOOST_PREDEF_MAKE_10_VVRRP)
+    #define BOOST_PREDEF_MAKE_10_VVRRP(V) BOOST_VERSION_NUMBER(((V)/1000)%100,((V)/10)%100,(V)%10)
+#endif
+
 //-----------------------------------------------------------------------------
 // clang CUDA compiler detection
 // Currently __CUDA__ is only defined by clang when compiling CUDA code.
-#if defined(__clang__) && defined(__CUDA__)
-    #define BOOST_COMP_CLANG_CUDA BOOST_COMP_CLANG
-#else
-    #define BOOST_COMP_CLANG_CUDA BOOST_VERSION_NUMBER_NOT_AVAILABLE
+#if !defined(BOOST_COMP_CLANG_CUDA)
```

Comment:
How can this ever happen? This macro has not been ported to boost so there is no one who could define this.

Reply:
well spotted. I guess this removal occurred with the changes due to `include/alpaka/core/BoostPredef.hpp`. Probably assumed it went there like the other definitions which can be found there.

Reply:
wait, the definition is there:
https://github.com/ComputationalRadiationPhysics/alpaka/blob/bebc38dc82b86c03a1edeba147a460c0e11f17c6/include/alpaka/core/Common.hpp#L41-L49

Reply:
I do not understand what you mean. It is the place I added the comment on. It should be possible to simply revert your change here.

Reply:
ah I read it like the definition would be gone (diff shows it as removed), but you actually only mean the
 #if !defined(BOOST_COMP_CLANG_CUDA)`, that this is not required, right?

Reply:
yes

Reply:
k, I'll remove that :)